test/deploy_veba_openfaas_processor.sh

```diff
@@ -31,6 +32,13 @@ VEBA_VCENTER_USER="administrator@vsphere.local"
 VEBA_VCENTER_PASS="VMware1!"
 VEBA_VCENTER_DISABLE_TLS="True"
 VEBA_OPENFAAS_PASS="VMware1!"
+VEBA_HOSTNAME="veba.primp-industries.com"
+VEBA_PREFIX="24 (255.255.255.0)"
+VEBA_GW="192.168.30.1"
+VEBA_DNS="192.168.30.1"
+VEBA_DNS_DOMAIN="primp-industries.com"
+VEBA_NTP="pool.ntp.org"
+VEBA_OS_PASSWORD="VMware1!"
```

Comment:
Looks like dupe entries were added to the OF example script, can you remove this?

Reply:
Oops.